frontend/src/utils/util.js

```diff
@@ -0,0 +1,7 @@
+export const titleCase = (str, delimiter = " ") => {
```

Comment:
I should use another file name for this logic ... something like string.js
and perhaps we could move current src/utils/trim.js into this new string.js file